specification/cognitiveservices/data-plane/CustomSearch/v1.0/examples/SuccessfulQueryRequest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "X-BingApis-SDK": "true",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "q": "cortana"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        
+      },
+      "body": {
+        "_type": "SearchResponse",
+        "webPages": {
+          "webSearchUrl": "https://www.bing.com/search?q=cortana",
+          "totalEstimatedMatches": 1,
+          "_type": "Web/WebAnswer",
+          "value": [
+            {
+              "id": "https://api.cognitive.microsoft.com/api/v7/#WebPages.0",
+              "name": "Cortana Skills - microsoft.com",
+              "url": "https://www.microsoft.com/en-us/cortana/skills/all",
+              "displayUrl": "https://www.microsoft.com/en-us/cortana/skills/all",
+              "snippet": "“Hey Cortana, ask Akinator to guess who I am thinking of.” “Hey Cortana, ask Akinator to start.” “Hey Cortana, ask Akinator to play a new round.” “Hey ...",
+              "dateLastCrawled": "2017-11-27T14:12:00.0000000Z"
+            }
```

Comment:
This model also needs a discriminator (very deep inside, one of is base models requires a discriminator property)